Tests/scripts/lock_cloud_machines.py

```diff
@@ -0,0 +1,359 @@
+                               lock_repository_name: str, machines_lock_repo: str):
+    """
+    get a list of all machines locks files.
+    Args:
+        storage_client(storage.Client): The GCP storage client.
+        bucket_name(str): the bucket name.
+        lock_repository_name(str):  the lock_repository_name name.
+        machines_lock_repo(str): the machines_lock_repo name.
+
+    Returns: list of dicts with the job-id and the time_created of the lock file.
+
+    """
+    blobs = storage_client.list_blobs(bucket_name)
+    files = []
+    found = False
+    for blob in blobs:
+        if blob.name.startswith(lock_repository_name):
+            found = True
+            if blob.name.startswith(f'{lock_repository_name}/{machines_lock_repo}'):
+                lock_file_name = blob.name.split('/')[-1]
+                if lock_file_name:
+                    files.append({'machine_name': lock_file_name.split('-lock-')[0],
+                                  'job_id': lock_file_name.split('-lock-')[1]})
+        elif found:
+            break
+    return files
+
+
+def check_job_status(token: str, job_id: str):
+    """
+    get the status of a job in gitlab.
+    Args:
+        token(str): the gitlab token.
+        job_id(str): the job id to check.
+    Returns: the status of the job.
+
+    """
+    user_endpoint = f"https://code.pan.run/api/v4/projects/2596/jobs/{job_id}"  # disable-secrets-detection
+    headers = {'PRIVATE-TOKEN': token}
+    response = requests.get(user_endpoint, headers=headers)
+    return response.json().get('status')
+
+
+def remove_build_from_queue(storage_bucket: any, lock_repository_name: str, job_id: str):
+    """
+    deletes a lock queue file
+    Args:
+        storage_bucket(google.cloud.storage.bucket.Bucket): google storage bucket where lock machine is stored.
+        lock_repository_name(str): the lock repository  name.
+        job_id(str): the job id to delete.
+    """
+    file_path = f'{lock_repository_name}/{QUEUE_REPO}/{job_id}'
+    blob = storage_bucket.blob(file_path)
+    try:
+        blob.delete()
+    except Exception as err:
+        logging.debug(f'when we try to delete a build_from_queue = {file_path}, we get an error: {str(err)}')
+        pass
+
+
+def remove_machine_lock_file(storage_bucket: any, lock_repository_name: str, machine_name: str, job_id: str):
+    """
+    deletes a lock machine file
+    Args:
+        storage_bucket(google.cloud.storage.bucket.Bucket): google storage bucket where lock machine is stored.
+        lock_repository_name(str) the lock repository name.
+        machine_name: the machine to delete
+        job_id(str): the job id to delete.
+    """
+    file_path = f'{lock_repository_name}/{machine_name}-lock-{job_id}'
+    blob = storage_bucket.blob(file_path)
+    try:
+        blob.delete()
+    except Exception as err:
+        logging.debug(f'when we try to delete a lock machine file = {file_path}, we get an error: {str(err)}')
+        pass
```

Comment:
It is the same function with a different parameter (file_path). Please create one function that removes file, and just call it with a different `file_path` argument.